files/image_config/pcie-check/pcie-check.sh

```diff
@@ -54,7 +54,7 @@ function check_and_rescan_pcie_devices()
 
      done
      debug "PCIe check failed"
-     redis-cli -n 6 HSET $PCIE_STATUS_TABLE "status" "FAILED"
```

Comment:
Is this change also needed for the reasons mentioned in the description? Or is this added for completeness (make all sonic-db-cli calls) reasons?

Reply:
This might have a very slight performance improvement. not significant i think. So i'd put this in the completeness category